README.md

```diff
@@ -82,7 +82,9 @@ The following users have deployed JanusGraph in production.
 * [Seeq](https://seeq.com)
 * [Sift Security](https://siftsecurity.com)
 * [Uber](https://uber.com)
+* [Flux.io](https://flux.io)
 
 The following companies offer JanusGraph hosted as-a-service:
 
 * [IBM](https://www.compose.com/databases/janusgraph)
+* [Flux.io](https://flux.io) - For AEC Industry Design & Construction Services
```

Comment:
I don't think you're offering JanusGraph as a service in the same sense as IBM Compose is — I presume you're offering design and construction SaaS, but you're not exposing JanusGraph directly to your users/customers, is that correct?

Reply:
Any update @FluxMcarrier ? Line 90 needs to be removed before merging.

Reply:
@FluxMcarrier https://flux.io website says at the top _FLUX Services will shut down after March 31st. [Read the press release.](https://builtworlds.com/news/breaking-news-flux-shifts-direction/)_

Will Flux still be a JanusGraph users with this change in direction?

Reply:
@FluxMcarrier — any updates on either of the two points above?